### DISCUSSION TITLE
Attactments in outgoing email as email attachments, not as links to rails.

### DISCUSSION THREAD

**Discussion by andrsharov** [chatwoot/chatwoot#10417] [Self hosted]
Hi, there

Chatwoot 3.14.1

We are using for backend S3 Compatibly Storage. All work fine.

Setting in `.env` file looks like: 
```
# Storage
ACTIVE_STORAGE_SERVICE=s3_compatible

# Amazon S3
# documentation: https://www.chatwoot.com/docs/configuring-s3-bucket-as-cloud-storage
STORAGE_BUCKET_NAME=******
STORAGE_ACCESS_KEY_ID=**********
STORAGE_SECRET_ACCESS_KEY=***********
STORAGE_REGION=**********
STORAGE_ENDPOINT=***********
```
But some clients, each works in big companies or goverment companies, don't have access to the internet.
Usually they only have Microsoft Outlook or Mozilla ThunderBird email clients and local phone.

How can I setup Chatwoot, to send outgoing email with email attachments as email attachments, not as links ?

At the same time, I want to save the attachment storage setting (incoming and outgoing) on our server in the S3 bucket.

**Comment by sojan-official** [comment]
fixed by this : https://github.com/chatwoot/chatwoot/pull/11304
